### ISSUE TITLE
Add functionality to print pipeline names

### ISSUE THREAD

**Issue by ChloeRN** [SPI-Birds/pipelines#135]
At the moment, run_pipelines() runs all pipelines in sequence. If one pipeline fails to run, the execution is halted. However, one then has to go "count" the bars of previously run pipelines to find out which pipeline failed. 
I suggest adapting code of **all** pipelines to print something like "**Running XXXX pipeline**:" when they start running. If possible, print it in bold too so it can more easily be distinguished from the other pipeline updates that are printed.

**Comment by LiamDBailey** [comment]
Agreed. I was thinking about this too! I'll work on this and can use the 'crayon' package to print messages with colours to hopefully make it clearer
